front-end/src/index.tsx

```diff
@@ -32,17 +34,23 @@ const httpLink = new HttpLink({
 const tokenRefreshLink = new TokenRefreshLink({
 	accessTokenField: 'token',
 	fetchAccessToken: getRefreshedToken,
-	handleError: (err:any) => {
-		console.error('There has been a problem with your fetch operation: ', err);
+	handleError: (err:Error) => {
+		deleteLocalStorageToken();
+		console.error('There has been a problem getting a new access token: ', err);
+		// FIXME probably redirect user to login with an error "you've been logged out"?
```

Comment:
This is where I'd like to use `history` to redirect users.. but I can't in a nice way as I can't use hooks here.

Reply:
noted